dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -617,7 +618,7 @@
       <PRIResource Remove="@(PRIResource)" />
     </ItemGroup>
     <!-- Use CreateItem to expand all the ** to individual full paths -->
-    <CreateItem Include="@(PRIResourceExpanded -> '%(Replaced)')">
+    <CreateItem Include="@(PRIResourceExpanded -&gt; '%(Replaced)')">
```

Comment:
**-&gt** what is this change?:) and it's repeating below several times

Reply:
Visual studio tends to do this unfortunately.

---
In reply to: [587828088](https://github.com/microsoft/microsoft-ui-xaml/pull/4403#discussion_r587828088) [](ancestors = 587828088)

Reply:
reverted